### ISSUE TITLE
How did deep supervision applied to the direction of arrival estimation?

### ISSUE THREAD

**Issue by eingkim** [CARNIVAL-IITP/Sound_source_localization#5]
How did deep supervision applied to the direction of arrival estimation?

**Comment by BaekMS** [comment]
The DNN model outputs the several spatial spectrums with each layer. Binary cross entropy loss between the estimated spatial spectrum and the different target spatial spectrum for each output is calculated. The loss is backpropagated to the DNN model.
